04 Strategy Library/999 Seasonality and Session Effect in Index Futures/01 Introduction.html

```diff
@@ -0,0 +1,7 @@
+<p>
+	In this research, we investigate the Seasonality and Market Session effect in the E-Mini S&P 500 futures. A recent article, Market Seasonality Study
+, argues that there is a seasonality effect in the futures market, specifically relating to E-Mini S&P 500 futures. The study argues that it's trading strategy implemented in the period from November to May lends itself to a bull market (Bullish Season), while the trading days between June and October (Bearish Season) hold far less profit. 
```

Comment:
Sorry I didn't initially catch this, but the Market Seasonality Study argues for the effect in the S&P as a whole (futures doesn't come into play until the second article)